shell/etc/csh/login.d/conda.csh

```diff
@@ -0,0 +1,43 @@
+
+# Recommended way to make the conda command available in csh is
+#   $ sudo ln -s <CONDA_ROOT>/etc/csh/login.d/conda.sh /etc/csh/login.d/conda.sh
```

Comment:
```
s/conda.sh/conda.csh
```
I am not sure, but shouldn't this be `/etc/profile.d/conda.csh` ? That's where `/etc/csh.login` sources stuff from.

Reply:
Really? Must be different on every distribution. That sucks. I think I looked at csh on Ubuntu. Guess I’ll check a couple different distros and see if I can tease out what’s most common

Reply:
Ok, on Ubuntu, `csh` doesn't pre-populate any files in `/etc`, but `tcsh` does.  Maybe that's the difference I'm seeing...

Reply:
```
root@47cf55bec20a:/# apt-get install csh
...
root@47cf55bec20a:/# echo "echo hello world" > /etc/profile.d/hello.csh
root@47cf55bec20a:/# . !$
. /etc/profile.d/hello.csh
hello world
root@47cf55bec20a:/# csh
#
```

I guess that didn't work.